Packs/GettingStartedWithXSOAR/pack_metadata.json

```diff
@@ -5,7 +5,7 @@
     "videos": [
         "https://www.youtube.com/watch?v=03TmGeAzvOQ"
     ],
-    "currentVersion": "1.0.0",
```

Comment:
I don't think there's a need to update this pack, since it only contains a wizard
Instead, the marketplace entries for `xsiam` and `platform` should be removed from the `pack_metadata`
this will prevent the pack from appearing in the marketplace altogether.